torchbenchmark/models/fastNLP_Bert/cmrc2018_simulator.py

```diff
@@ -4,12 +4,10 @@
 """
 import os
 import pathlib
-import numpy
 import json
 import random
-import fastNLP
 
-TRAIN_BATCH_SIZE = 1
+TRAIN_BATCH_SIZE = 6
```

Comment:
It may be good to leave a comment here to claim that this number comes from the documentation/src code.

Reply:
Thanks! This is resolved.